components/Tabs/index.js

```diff
@@ -7,3 +7,20 @@
 //
 //  The tab component should look like this:
 //    <div class="tab">topic here</div>
+
+
+// Making API request
+axios.get('https://lambda-times-backend.herokuapp.com/topics')
+.then(staff => staff.data.topics.forEach(element => new Tab(element)))
```

Comment:
It is considered best practice to always chain a .catch function to promises in order to handle errors in case the promise is rejected